modules/runtime/nuxeo-launcher-commons/src/test/java/org/nuxeo/launcher/config/ConfigurationGeneratorTest.java

```diff
@@ -329,6 +330,23 @@ public void testIncludeProfile() throws Exception {
         assertEquals("true", generator.getUserConfig().getProperty("nuxeo.profile.added.by.test"));
     }
 
+    @Test
+    public void testIncludeTemplatesWildcard() throws Exception {
+        ConfigurationGenerator generator = generatorBuilder().init(true).build();
+        ConfigurationHolder configHolder = generator.getConfigurationHolder();
+        // test templates are all included
+        List<String> includedTemplates = configHolder.getIncludedTemplateNames();
+        assertTrue(includedTemplates.contains("default"));
+        assertTrue(includedTemplates.contains("test"));
+        assertTrue(includedTemplates.contains("mongodb"));
+        assertTrue(includedTemplates.contains("cloud-directupload"));
```

Comment:
Are these needed given the ordered assertion below?

Reply:
Indeed, let leverages the ordered assertion.